soc/arm/st_stm32/stm32f7/Kconfig.defconfig.stm32f745xx

```diff
@@ -0,0 +1,14 @@
+# ST STM32F745XX MCU configuration options
+
+# Copyright (c) 2020 Kwon Tae-young <tykwon@m2i.co.kr>
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_STM32F745XX
+
+config SOC
+	default "stm32f745xx"
+
+config NUM_IRQS
+	default 97
```

Comment:
98

Reply:
Should I add 1 to this?
![image](https://user-images.githubusercontent.com/10510127/84127802-ab9ac380-aa7a-11ea-91c5-13c1c4b4c4ec.png)

Reply:
@KwonTae-young theoretically, you shouldn't.

But practically the rule applied is to use max irq number (97 here) + 1 (as numbering starts from 0). Number is taken from https://raw.githubusercontent.com/zephyrproject-rtos/hal_stm32/master/stm32cube/stm32f7xx/soc/stm32f745xx.h enum.
I prefer this practial rule that is easy to verify and to explain.

In either case, aim is to define size of the isr_table that is generated at compile time. I don't think this should matter as I doubt it could be possible to reach the max anyway.
Anyway, we should eventually get of rid of this and be able to define the size at compile time based on device tree.